uvicorn/config.py

```diff
@@ -102,8 +102,14 @@
 
 
 def create_ssl_context(
-    certfile, keyfile, password, ssl_version, cert_reqs, ca_certs, ciphers
-):
+    certfile: Optional[str],
```

Comment:
`certfile` is a required positional argument when running [`SSLContext.load_cert_chain`](https://docs.python.org/3/library/ssl.html#ssl.SSLContext.load_cert_chain), so annotating as `Optional` (which allows `None`) would not be ideal here. If path-like objects are acceptable, after `from pathlib import Path`, the annotation could be `certfile: Union[Path, str]`.